src/Network/Network/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 7.22.1
```

Comment:
LGTM but please don't put a version in `ChangeLog.md` because versions are calculated.
@DanielMicrosoft would you mind editing it for the author?